modules/internal/String.chpl

```diff
@@ -114,19 +114,22 @@ units for offsets or lengths:
  * codepoints
  * graphemes
 
-Most methods on the Chapel string type currently work with byte units by
-default. For example, :proc:`~string.length` returns the length in bytes and
-`int` values passed into :proc:`~string.this` are offsets in byte units.
+Most methods on the Chapel string type currently work with codepoint units by
+default. For example, :proc:`~string.length` returns the length in codepoints
+and `int` values passed into :proc:`~string.this` are offsets in codepoint
+units.
 
-The string type currently supports codepoint units as well. In some cases these
-units are indicated by the method name, for example with
-:proc:`~string.numCodepoints`. Other methods, including :proc:`~string.this`, support
-arguments of type :record:`codepointIndex` (instead of `int`) in order to
-request codepoint units.
+The string type currently supports byte units as well. Most methods,
```

Comment:
"The string type currently supports byte units as well" -> "It is possible to indicate byte or codepoint units for indexing in the string methods."